p2p/src/sync/peer.rs

```diff
@@ -388,6 +578,8 @@ where
     /// This is needed to allow the local or remote node to have slightly inaccurate clocks.
     /// Without it, even a 1 second difference can break block synchronization
     /// because one side may see the new block as invalid.
+    // TODO: this must be removed; but at this moment this is not really possible, because
```

Comment:
Please reference this issue: https://github.com/mintlayer/mintlayer-core/issues/1024